manim/mobject/geometry/line.py

```diff
@@ -1015,6 +1015,53 @@ def construct(self):
             return self.angle_value / DEGREES
         return self.angle_value
 
+    @staticmethod
+    def from_three_points(
+        A: np.ndarray, B: np.ndarray, C: np.ndarray, **kwargs
+    ) -> Angle:
+        """The angle between the lines AB and BC.
```

Comment:
```suggestion
        r"""The angle between the lines AB and BC.
```

This has to be a raw string because there is a backslash in the body. (Otherwise \a is interpreted as one character as opposed to \ and a.)